final_task/rss_reader/classes/news_class.py

```diff
@@ -38,8 +36,8 @@ def create_string_of_links(self):
         string_repr_of_links = ''
         for num, link in enumerate(list_of_links):
             if link:
-                string_repr_of_links = string_repr_of_links + '[{}] '.format(num + 1) + link + '\n'
-        self.links = string_repr_of_links
+                string_repr_of_links = string_repr_of_links + "[{}] {}\n".format(num+1, link)
```

Comment:
It is still beter to use `formatetd string` here: `f"[{num+1}] {link}"`

Reply:
Fixed